01-intro.Rmd

```diff
@@ -42,7 +42,7 @@ At its most basic level Git allows the following:
 
 3. **Merge** - Systematically merge these changes back into the main version of the code.
 
-![](www/branching_diagram.png)
+![A diagram to demonstrate branching with Git.](www/images/chapter1_branching_diagram.png)
```

Comment:
Is there a reference for this image like you have included with the others?

Reply:
I didn't put this image in and a reverse image search doesn't give me any results for the source. Any ideas where this picture has come from?